src/test.ts

```diff
@@ -122,31 +154,43 @@ export function consoleTestResultHandler(testResult: TestResult): boolean {
         const results = testResult.results[fileName];
         process.stdout.write(`${fileName}:`);
 
-        const diffResults = diff.diffLines(results.markupFromMarkup, results.markupFromLinter);
-        const didTestPass = !diffResults.some((diff) => diff.added || diff.removed);
+        const markupDiffResults = diff.diffLines(results.markupFromMarkup, results.markupFromLinter);
+        const fixesDiffResults = diff.diffLines(results.fixesFromMarkup, results.fixesFromLinter);
+        const didMarkupTestPass = !markupDiffResults.some((diff) => diff.added || diff.removed);
+        const didFixesTestPass = !fixesDiffResults.some((diff) => diff.added || diff.removed);
 
         /* tslint:disable:no-console */
-        if (didTestPass) {
+        if (didMarkupTestPass && didFixesTestPass) {
             console.log(colors.green(" Passed"));
         } else {
             console.log(colors.red(" Failed!"));
-            console.log(colors.green(`Expected (from ${FILE_EXTENSION} file)`));
-            console.log(colors.red("Actual (from TSLint)"));
-
             didAllTestsPass = false;
-
-            for (const diffResult of diffResults) {
-                let color = colors.grey;
-                if (diffResult.added) {
-                    color = colors.green;
-                } else if (diffResult.removed) {
-                    color = colors.red;
-                }
-                process.stdout.write(color(diffResult.value));
+            if (!didMarkupTestPass) {
+                displayDiffResults(markupDiffResults, MARKUP_FILE_EXTENSION);
+            }
+            if (!didFixesTestPass) {
+                displayDiffResults(fixesDiffResults, FIXES_FILE_EXTENSION);
             }
         }
         /* tslint:enable:no-console */
     }
 
     return didAllTestsPass;
 }
+
+function displayDiffResults(diffResults: diff.IDiffResult[], extension: string) {
+    /* tslint:disable:no-console */
+    console.log(colors.green(`Expected (from ${extension} file)`));
+    console.log(colors.red("Actual (from TSLint)"));
```

Comment:
Will we want any message here to differentiate that you're looking at fix or lint failure differences?